README.md

```diff
@@ -258,18 +257,26 @@ You can show the last date a page was last modified by uncommenting this line fr
 This will add the date right before the footer component and uses the `last-modified.html` include.
 
 ```yml
-# Used to show the "Last updated" date and time;
 # last_updated: true
 ```
 
+### Anchor JS
+
+You can show an anchor link next to header tags by uncommenting this section from the `_config.yml` data file.
```

Comment:
Maybe a note about what this is actually for would be good. 

"Anchor links will make individual section headers linkable" or something to that effect.

Reply:
Yes, great idea!